crates/astria-grpc-mock/src/mock.rs

```diff
@@ -0,0 +1,147 @@
+use std::ops::{
```

Comment:
would be nice to have a small doc comment on usage (although looks similar to wiremock?)

Reply:
Will do, but leave it to a follow up. https://github.com/astriaorg/astria/issues/901